packages/vertexai/src/public-types.ts

```diff
@@ -21,20 +21,136 @@ export * from './types';
 
 /**
  * An instance of the Vertex AI in Firebase SDK.
+ *
+ * For more information, refer to the documentation for the new {@link GenAI}.
```

Comment:
"refer to the documentation for the new {@link GenAI} interface"? So they know it's a new API surface and not a new product or something.

Reply:
good point, done